projects/sandbox/datagen/datagen/scripts/background.py

```diff
@@ -208,15 +239,61 @@ def deploy(
     logdir: Path,
     accounting_group: str,
     accounting_group_user: str,
-    # note that doing consecutive runs while changing max_segment_length
-    # will screw with the caching checking, so be careful
     max_segment_length: float = 20000,
     request_memory: int = 32768,
     request_disk: int = 1024,
     force_generation: bool = False,
     verbose: bool = False,
 ):
+    """
+    Deploy condor jobs to download background data.
 
+    Args:
+        train_start:
+            GPS time of the beginning of the training period.
+        train_stop:
+            GPS time of the end of the training period.
+            Also corresponds to the beginning of the testing period.
+        test_stop:
+            GPS time of the end of the testing period.
+        minimum_train_length:
+            The shortest a contiguous segment of training background can be.
+            Specified in seconds.
+        minimum_test_length:
+            The shortest a contiguous segment of testing background can be.
+            Specified in seconds.
+        ifos:
+            List of interferometers to query data from. Expected to be given
+            by prefix; e.g. "H1" for Hanford
+        sample_rate:
+            Sample rate to which the timeseries will be resampled
+        channel:
+            Channel from which to fetch the timeseries
+        state_flag:
+            Identifier for which segments to use
```

Comment:
Could be good to link to LIGO wiki documentation about what this means and what the available flags are.